docs/getting-started/introduction.md

```diff
@@ -61,6 +62,7 @@ Select your state and the consuming component
 will re-render when that state changes.
 
 ```jsx
+import { useStore } from './store';
```

Comment:
```suggestion
import { useStore } from './store';

```

Reply:
i think you want me to remove the import of useStore in the component

Reply:
No, just add a line after the import